scripts/jenkins/qa_openstack.sh

```diff
@@ -365,7 +365,15 @@ outputs:
   server_floating_ip:
     value: { get_attr: [ my_floating_ip, floating_ip_address ] }
 EOF
+
+os_client_ret=1
+
 if [ -x /usr/bin/openstack ]; then
+    openstack stack list > /dev/null 2>&1
```

Comment:
I'm not convinced that this is a good test. I would like to see a test based on $cloudsource or on "openstack --version" or something like that instead. otherwise a failed stack list (due to a server side error) might cause it to switch back to heatclient.